pipelines/fast-test.yml

```diff
@@ -87,6 +87,10 @@ stages:
     variables:
       YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
     steps:
+    - task: NodeTool@0
```

Comment:
Fast test will break without this? How about full test?

Reply:
I guess they will not be affected. Because it's pretty much `UsePythonVersion` of node.js, and we don't have `UsePythonVersion` in integration tests.